stable/database/README.md

```diff
@@ -317,17 +302,19 @@ The purpose of this section is to allow customisation of the names of the cluste
 Verify the Helm chart:
 
 ```bash
-helm install nuodb/database -n database \
+helm install nuodb/database --name database \
     --debug --dry-run
 ```
 
 Deploy a database without backups:
 
 ```bash
-helm install nuodb/database -n database \
+helm install nuodb/database --name database \
     --set database.sm.hotcopy.replicas=0 --set database.sm.nohotcopy.replicas=1
 ```
 
+**Note**: If deploying in Red Hat OpenShift add `--set openshift.enabled=true` to your `helm install` command.
+
```

Comment:
this is not true. An SCC needs to be installed. We are not using the openshift.enabled flag.

Reply:
Thanks for confirming. I likely was using this prior to the adding of the scc yaml file add. Removed.